### PR TITLE
Connected game over scene to main menu with some fixes along the way.

### PR DESCRIPTION
**Pull request by Schtekt**

### CODE REVIEW

**Review by AntonAsbrink — DISMISSED** [review]
Currency återställs inte till 0 efter en gameover

File: StortSpelProjekt/Game/src/main.cpp [hunk 1]
```diff
@@ -842,6 +808,14 @@ Scene* ShopScene(SceneManager* sm)
...
+{
+    QuadManager* man = scene->GetEntity("healthbar")->GetComponent<component::GUI2DComponent>()->GetQuadManager();
+    scene->GetEntity("healthbar")->GetComponent<component::GUI2DComponent>()->GetQuadManager()->UpdateQuad(man->GetPos(), { 0.275f, 0.055f }, false, false,
+        { 1.0, 1.0, 1.0, 1.0 },
+        float3{ 0.0f, 1.0f, 0.0f });
+}
+
 void GameUpdateScene(SceneManager* sm, double dt)
 {
     if (ImGuiHandler::GetInstance().GetBool("reset"))
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Game/src/main.cpp:841]: Maybe call the reset functions here? 😁

**Review by LeoWikstrom — DISMISSED** [review]
Works well. One thing about the Game Over scene though: The Exit button should be further to the right and seven pixels (on my screen at least) further down

**Review by LeoWikstrom — DISMISSED** [review]
👌🏽

**Review by AntonAsbrink — CHANGES_REQUESTED** [review]
Upgrades or shop does not reset through runs

**Review by LeoWikstrom — DISMISSED** [review]

**Review by LeoWikstrom — APPROVED** [review]

**Review by AntonAsbrink — APPROVED** [review]

**Pull request merged**